includes/class-newspack-popups.php

```diff
@@ -388,5 +389,28 @@ public static function popup_default_fields( $post_id, $post, $update ) {
 		update_post_meta( $post_id, 'trigger_scroll_progress', 30 );
 		update_post_meta( $post_id, 'utm_suppression', '' );
 	}
+
+	public static function create_lightweight_api_config() {
+		$config_path = WP_CONTENT_DIR . '/../newspack-popups-config.php';
+		$has_config  = file_exists( $config_path );
+		if ( false === $has_config ) {
+			global $wpdb;
+			$config_file = fopen( $config_path, 'w' );
+			if ( $config_file ) {
+				fwrite(
```

Comment:
This is perfect as is and no need to change, just sharing that I usually prefer to use `file_put_contents()` instead, because it's a bit simpler -- doesn't need `fopen` and `fclose`, and takes file name string as argument.